src/sudoers/mod.rs

```diff
@@ -132,6 +165,38 @@ impl Sudoers {
     }
 }
 
+fn group_cmd_specs_per_runas<'a>(
+    list: impl Iterator<Item = (Option<&'a RunAs>, (Tag, &'a Spec<Command>))>,
+    entries: &mut Vec<Entry<'a>>,
+) {
+    static EMPTY_RUNAS: RunAs = RunAs {
```

Comment:
I think this can be a regular `const` can't it?

Reply:
`const` would also work but semantically `static` feels more correct as we'll create `&'static` references to it (to the *same memory location*) below

`const` is not a fixed memory location but instead a value that's "inlined" wherever is used. due to the "static promotion" pass in the compiler you can create `&'static` references out of it but that could (would?) be creating a new static at each `&CONSTANT` location which then the optimizer has to deduplicate. a `static` variable avoids relying on the optimizer